lib/archethic/contracts/contract/context.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Archethic.Contracts.Contract.Context do
```

Comment:
I didn't find any serialization / deserialization of the Context used in NewTransaction / StartMining message

Reply:
wow! this means all my tests were wrong and useless. I'll add missing tests, fix this and retest everything